semgrep-core/src/analyzing/Dataflow_tainting.ml

```diff
@@ -249,16 +255,23 @@ let (transfer :
 (* Entry point *)
 (*****************************************************************************)
 
+(* Very silly function. We previously relied on the fact that F.cfg and X.flow were equal types,
+   but because OCaml does not have first class records, this equivalence no longer holds definitionally.
+   Instead we have to manually convert between the types even though they are technically the same
+*)
+let cfg_to_flow ({ graph; entry } : F.cfg) : X.flow = { graph; entry }
```

Comment:
As I said in a previous comment, I think we should have a `('node, 'edge) cfg` type in `Dataflow` and everywhere else we just instantiate that type.